<launch>
  <rosparam file="$(find generate_tf)/param/static_tf.yaml" command="load" ns="static_tf"/>
  <rosparam file="$(find generate_tf)/param/static_tf_index.yaml" command="load" ns="static_tf"/>

  <rosparam file="$(find generate_tf)/param/tf_broadcastor.yaml" command="load" ns="tf_broadcastor"/>
  <rosparam file="$(find generate_tf)/param/tf_broadcastor_index.yaml" command="load" ns="tf_broadcastor"/> 

 <node pkg="generate_tf" name="static_broadcastor_node" type="static_broadcastor_node" output="screen"></node>
  <node pkg="generate_tf" name="tf_broadcastor_node" type="tf_broadcastor_node" output="screen"></node> 
</launch>
